# Question 3

## (a)

In [19]:
library(faraway)

In [20]:
data(teengamb)

In [18]:
linear_model<-lm(gamble~sex+status+income+verbal,data=teengamb)

In [21]:
summary(linear_model)


Call:
lm(formula = gamble ~ sex + status + income + verbal, data = teengamb)

Residuals:
    Min      1Q  Median      3Q     Max 
-51.082 -11.320  -1.451   9.452  94.252 

Coefficients:
             Estimate Std. Error t value Pr(>|t|)    
(Intercept)  22.55565   17.19680   1.312   0.1968    
sex         -22.11833    8.21111  -2.694   0.0101 *  
status        0.05223    0.28111   0.186   0.8535    
income        4.96198    1.02539   4.839 1.79e-05 ***
verbal       -2.95949    2.17215  -1.362   0.1803    
---
Signif. codes:  0 ‘***’ 0.001 ‘**’ 0.01 ‘*’ 0.05 ‘.’ 0.1 ‘ ’ 1

Residual standard error: 22.69 on 42 degrees of freedom
Multiple R-squared:  0.5267,	Adjusted R-squared:  0.4816 
F-statistic: 11.69 on 4 and 42 DF,  p-value: 1.815e-06


### As seen in the summary of the nested linear model, sex and income are statiscally significant at the 5% level and their 95% confidence intervals are given below.

In [22]:
confint(linear_model)[c('sex','income'),]

2.5 %      97.5 %   
sex    -38.689030 -5.547630
income   2.892654  7.031305

## (b)

In [23]:
linear_model_2<-lm(gamble~income,data=teengamb)

In [24]:
anova(linear_model_2,linear_model)

Res.Df RSS      Df Sum of Sq F        Pr(>F)    
1 45     28008.59 NA       NA        NA         NA
2 42     21623.77  3 6384.821  4.133761 0.01177211

### As seen by the ANOVA test, the three extra variables in the full model give more information about the data than using only income as a predictor. This is because the F-statistic is 4.134 which gives the p-value of 0.0117, thus making the extra three predictors statiscally significant.

## (c)

### Null Hypothesis $H_{0}$ : $\beta_{sex}$=$\beta_{income}$=$\beta_{verbal}$=$\beta_{status}$=0
### Alternative Hypothesis $H_{1}$ : at least one of $\beta_{sex}$,$\beta_{income}$,$\beta_{verbal}$,$\beta_{status}$$\neq$0

# Question 4

In [25]:
(t.stat <- 0.7190/0.1535)

[1] 4.684039

In [26]:
(p.val <- 2*pt(-t.stat, 28))

[1] 6.577634e-05

In [27]:
RSS <-(0.8662^2)*28
TSS <-21.94*(RSS/28) + RSS
R.squared <- 1 - (RSS/TSS)
R.squared

[1] 0.4393272

### The three missing entries are: t-statistic=$4.684$, p-value=$6.578*10^-{5}$ and Multiple R-Squared=$0.439$